ERKER2Phenopackets/data/out/phenopackets/example-phenopackets-from-synthetic-data/0.json

```diff
@@ -55,7 +71,7 @@
     }
   ],
   "metaData": {
-    "created": "2023-09-22T00:00:00Z",
+    "created": "2023-09-26T00:00:00Z",
     "createdBy": "P. Robinson, MD, D. Danis, PhD, A. Graefe, F. Rehburg",
```

Comment:
@frehburg @graefea it's flattering but I don't think there is any need to add me and Peter here at this point. :)